config/kernel-acl.m4

```diff
@@ -230,7 +230,22 @@ dnl #
 dnl # 5.12 API change,
 dnl # set_acl() added a user_namespace* parameter first
 dnl #
+dnl # 6.2 API change,
+dnl # set_acl() second paramter changed to a struct dentry *
+dnl #
```

Comment:
Can you add a similar comment above `ZFS_AC_KERNEL_SRC_INODE_OPERATIONS_GET_ACL`.  This was also overlooked when commit 884a69357fe47ef317085bd05fb197a7f321e3f4 was merged.

Reply:
No problem, updated to add this at Line 168 in `config/kernel-acl.m4`